etc/ci/buildbot_steps.yml

```diff
@@ -23,8 +23,8 @@ mac-rel-css:
   - bash ./etc/ci/manifest_changed.sh
 
 linux-dev:
-  - ./mach test-tidy --no-progress
   - ./mach test-tidy --no-progress --self-test
+  - ./mach test-tidy --no-progress --all
```

Comment:
Let's keep the `--all` line first, to match https://github.com/servo/saltfs/blob/master/buildbot/master/files/config/steps.yml.